src/lava/lib/optimization/utils/solver_tuner.py

```diff
@@ -26,37 +27,41 @@ def __init__(self, params_grid: ty.Dict):
     def tune(self,
              solver,
              solver_parameters: ty.Dict,
-             stopping_condition: ty.Callable[[float, int], bool] = None):
-        """Find and set solving hyperparameters for an OptimizationSolver"""
+             stopping_condition: ty.Callable[[float, int], bool] = None,
+             seed: int = 0):
+        """
+        Perform random search to optimize solver hyper-parameters.

```

Comment:
For docstrings, we encourage to describe all input and output parameters in the following form:

![image](https://user-images.githubusercontent.com/86950058/199703316-451ae380-ef06-47de-952a-c8224b38ff1f.png)